account_chatter/models/account_move.py

```diff
@@ -6,5 +6,5 @@ class AccountMove(models.Model):
 
     date = fields.Date(tracking=True)
     journal_id = fields.Many2one(tracking=True)
-    narration = fields.Text(tracking=True)
+    narration = fields.Html(tracking=True)
```

Comment:
Fields tracking is [not supported](https://github.com/odoo/odoo/blob/16.0/addons/mail/models/mail_tracking_value.py#L57) for HTML field, so just remove this inheritance.

Reply:
done